src/pages/category/Index.vue

```diff
@@ -1,43 +1,86 @@
  <template>
   <div>
-    <a-row :gutter="24">
+    <a-row :gutter="24" style>
       <a-col :span="6">
-        <a-input-search style="margin-bottom: 8px" placeholder="Search" @change="onChange" />
-        <a-tree
-          :expanded-keys="expandedKeys"
-          :auto-expand-parent="autoExpandParent"
-          :show-line="true"
-          :tree-data="categories"
-          :replaceFields="{children:'children', title:'name', key:'id' }"
-          @expand="onExpand"
```

Comment:
if `@expand="onExpand"` and `@change="onChange"` is no longer be used. the function defined in `methods` should be removed synchronously.